src/components/SocialShareButtons.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <div class="social-share">
+    <h5>Share</h5>
```

Comment:
Same here

Reply:
These are the changes after updation:

![Screenshot from 2020-03-31 18-35-13](https://user-images.githubusercontent.com/35831931/78029763-b9e3a980-737e-11ea-9ff2-e8372fdf02e2.png)
![Screenshot from 2020-03-31 18-35-19](https://user-images.githubusercontent.com/35831931/78029853-dbdd2c00-737e-11ea-9db8-686a6d04f28e.png)